uninstall.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "github.com/spf13/cobra"
+
+var Uninstall = &cobra.Command{
+	Use:    "CLI-MESSAGE-UNINSTALL",
+	Hidden: true,
+	Short:  "Uninstall",
+	Long:   "Runs any tasks required for a clean uninstall",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return nil
+	},
```

Comment:
As this currently doesn't do anything, is there any value in having it here at all?

Reply:
Yes. It is executed when the plugin gets uninstalled via `cf uninstall-plugin`.

It's a noop but a happy noop.

If the plugin returns a bad status to this during uninstall then the user gets a big scarey red failure message (even tho the uninstall completes ok).